index.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Technical Documentation</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+<nav id="navbar">
+    <header class="navbar-header">General Coding information</header>
+    <li><a class="nav-link" href="#HTML_and_CSS">HTML and CSS</a></li>
+    <li><a class="nav-link" href="#JavaScript">JavaScript</a></li>
+    <li><a class="nav-link" href="#Python">Python</a></li>
+    <li><a class="nav-link" href="#React">React</a></li>
+    <li><a class="nav-link" href="#APIs">APIs</a></li>
+</nav>
+
+<main id="main-doc">
+
+    <section id="HTML_and_CSS" class="main-section">
+        <header>HTML and CSS</header>
```

Comment:
- You should have only one `<header>` tag in your HTML file, instead you can have more `<h1>`, `<h2>`, `<h3>` ...
change your headers to `h` tags.